src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTryFinally.cs

```diff
@@ -2007,26 +2013,27 @@ public void ThrowInFinally()
 @"using System;
 class C
 {
+    static void nop() { }
     static void ThrowInFinally()
     {
-        try { }
+        try { nop(); }
         finally { throw new Exception(); }
     }
     static void ThrowInFinallyInTry()
     {
         try
         {
-            try { }
+            try { nop(); }
```

Comment:
no need to put nops here since we do not have catches. We also need a test to verify that nonempty finallies do not disappear, so this could be one such test if nops are removed.